### PR TITLE
added cache for local import paths

### PR DESCRIPTION
**Pull request by neelance**

### CODE REVIEW

File: build/build.go [hunk 1]
```diff
@@ -505,11 +505,19 @@ func (s *Session) BuildPackage(pkg *PackageData) (*compiler.Archive, error) {
 		return nil, err
 	}
 
+	localImportPathCache := make(map[string]*compiler.Archive)
```
Comment (shurcooL) [inline build/build.go]: Map key should be documented.

Comment (shurcooL) [inline build/build.go]: Unless you think `localImportPath` in the variable name is descriptive enough?

Comment (neelance) [inline build/build.go]: I think so.

Comment (shurcooL) [inline build/build.go]: So is it the full import path without /vendor/ stripped?

**Pull request merged**